warn/warn_test.go

```diff
@@ -1107,3 +1107,132 @@ native.cc_library(name = "lib")
 		`:1: "native.package()" shouldn't be used in .bzl files.`,
 	}, scopeBzl)
 }
+
+func TestMissingReturnValueWarning(t *testing.T) {
```

Comment:
Did you check the tests from Skylint?

Reply:
Yes, they are more or less similar to mine.